replay_build_scripts/lint.mjs

```diff
@@ -24,7 +24,7 @@ function findMatches(text /*: string*/, regex /*: RegExp*/) {
 function extractNamedScriptBlock(
   text /*: string*/,
   start /*: number*/,
-  end /*: number*/,
+  end /*: number*/
```

Comment:
This and some other changes in this file came from auto-format. We should lint this file, too 🤔

Reply:
we should really encode our formatting requirements in a way everyone's editor can use.  we don't (afaik?) have a shared prettier config, e.g.  so whoever saves the file last generates a new diff.